ios/Exponent/Versioned/Core/Api/SafeAreaContext/RNCSafeAreaUtils.h

```diff
@@ -0,0 +1,8 @@
+#import <React/RCTDefines.h>
+#import <React/RCTView.h>
+#import <UIKit/UIKit.h>
+
+extern NSString * const RNCSafeAreaDidChange;
+
+RCT_EXTERN BOOL
+UIEdgeInsetsEqualToEdgeInsetsWithThreshold(UIEdgeInsets insets1, UIEdgeInsets insets2, CGFloat threshold);
```

Comment:
I'm afraid this will have to be prefixed during versioning 🤔 We would have duplicated symbols otherwise.

Reply:
nice catch! the issue should be fixed by the latest commit.